protocol/protobuf/url_data.proto

```diff
@@ -30,10 +30,3 @@ message URLData {
  // Community, Channel, or User
  bytes content = 1;
```

Comment:
Another suggestion is to avoid second parsing [here](https://github.com/status-im/status-go/pull/3855/files#diff-5ec47bfa4a368489277346631ec15cec1e6460766c38267f77e6db8971a3ba0fR187):
```
message URLData {
optional Community community = 1;
optional Channel channel = 2;
optional User = 3;
```